test/fuzz/meson.build

```diff
@@ -36,18 +36,26 @@ fuzz_regression_tests += { 'fuzz-unit-file' : dict }
 
 ############################################################
 
+if get_option('auto_features').enabled()
+        sanitize_auto_features = 'enabled'
```

Comment:
Is it not possible to stringify auto_features?

Reply:
Turns out it's possible… with meson >= 1.3.0 :D

(I added a TODO comment.)